user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md

```diff
@@ -28,41 +27,63 @@ The predicted trend is displayed as a line surrounded by three shaded areas:
 
 > [!NOTE]
 >
-> - The probability percentages are based on the assumptions made at the moment the graph is generated. If the behavior of the data changes afterwards, the probability may need to be adjusted.
-> - If a recurring pattern is found, the predicted trend is based on this pattern. Otherwise, the predicted trend should be considered as the average future value.
+> - The predicted trend is generated by a machine learning model fit on the historical trend data of the metric.
```

Comment:
```suggestion
> - The predicted trend is generated by a machine learning model trained on the historical trend data of the metric.
```

I am unsure about this wording of a model "fit" on trend data. Would "trained" be more accurate?

Reply:
I am ok with changing it to "training"